styles/_base.scss

```diff
@@ -146,22 +147,18 @@ ul, ol {
   }
 }
 
-// temporary status bar implementation
-#statusBarContainer {
-  color: black; // use color var when refactoring
-  background-color: rgba(0,0,0,0.3); // use color var when refactoring
-  height: 20px;
-  position: fixed;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  font-size: 16px;  // use rem when refactoring
-  padding: 5px;
-
-  &::after {
-    content: '(temporary Walmart of status bars)';
-    position: absolute;
-    right: 5px;
-    top: 5px;
-  }  
+// turn off animations and transitions with this style
+
+#ov1.minEffects,
```

Comment:
We'll still need it, some Linux machines and old or low powered machines need the effects turned off to run well.